### PR TITLE
Add GitHub Actions Workflow for Code Checks

### PR DESCRIPTION
**Pull request by mayorjay**

### CODE REVIEW

File: .github/workflows/code-checks.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+jobs:
+  checks:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Set up Java
+        uses: actions/setup-java@v4
+        with:
+          java-version: '17'
```
Comment (enyason) [inline .github/workflows/code-checks.yml:18]: would the java version specified here become an issue, considering I set up the project with Java 1.8?

**Review by enyason — COMMENTED** [review]
LGTM

File: .github/workflows/code-checks.yml (hunk 1 above)
Comment (mayorjay) [reply .github/workflows/code-checks.yml:18]: No, it won't

Comment (enyason) [reply .github/workflows/code-checks.yml:18]: It's perfect then.

**Pull request merged**